app/controllers/signin.js

```diff
@@ -55,6 +57,10 @@ export default Controller.extend(ValidationEngine, {
 
                 this.set('flowErrors', (mainError.context.string || mainError.message.string));
 
+                if (mainError.message.string.match(/your password must be reset/i)) {
```

Comment:
We should be switching on the `PasswordResetRequiredError` that should be available on `mainError.type` if possible - messaging can change so it's brittle to inspect the message string.